nswag/Client.Class.liquid

```diff
@@ -385,7 +385,7 @@ Success is always expected
 {%         endif -%}
             {
                 var responseData_ = response_.Content == null ? null : await response_.Content.ReadAsStringAsync().ConfigureAwait(false);
-                throw new {{ ExceptionClass }}("The HTTP status code of the response was not expected (" + status_ + ").", status_, responseData_, headers_, null);
+                throw {{ ExceptionClass }}.Create(status_, headers_, responseData_, JsonSerializerSettings, null);
```

Comment:
This line handles the case when an undocumented status code is received. When creating the `ApiException` in this case, it will try to deserialize the responseData to a `ProblemDetails`. If unable to deserialize, then a `ProblemDetails` with minimal information is created. See this commit to view what changes this line made to the generated code.
https://github.com/Laserfiche/lf-repository-api-client-dotnet/pull/61/commits/e1359cd80baca790927633f9ba3e273bc0df11a4